src/components/input/Input.stories.tsx

```diff
@@ -2,13 +2,26 @@ import React from 'react';
 import { Story, Meta } from '@storybook/react/types-6-0';
 
 import { Input, InputProps } from './';
+import { LabelSmall } from 'baseui/typography';
 
 export default {
   title: 'Components/Input',
   component: Input,
 } as Meta;
 
-const Template: Story<InputProps> = (args) => <Input {...args}></Input>;
+const Template: Story<InputProps> = (args) => (
+  <div>
+    <Input {...args} />
+    <Input
+      placeholder="00:00"
+      startEnhancer={<LabelSmall overrides={{ Block: { style: { width: '100px' } } }}>start</LabelSmall>}
+    />
+    <Input
+      placeholder="00:00"
+      startEnhancer={<LabelSmall overrides={{ Block: { style: { width: '100px' } } }}>endendendend</LabelSmall>}
+    />
```

Comment:
These should be created as separate stories and props should be passed through `args` so that you can see which props are being passed in storybook.

I think for now we can just remove this. This is very `Caesar` specific, while this library aims to be general-purpose